docs/specifications/tests/MethodsV2.md

```diff
@@ -1,70 +1,92 @@
+* [Method: Get zone NS names and IP addresses][Get-Zone-NS-Names-and-IPs]
+* [Method: Get zone NS IP addresses][Get-Zone-NS-IPs]
+* [Method: Get in-bailiwick address records in zone (internal)][Get-IB-Addr-in-Zone]
+* [Method: Get delegation (internal)][Get-Delegation]
+* [Method: Get out-of-bailiwick ip addresses (internal)][Get-OOB-IPs]
+* [Method: Get data for undelegated test (internal)][Get-Undel-Data]
 * [Method inter-dependencies](#method-inter-dependencies)
 * [Terminology](#terminology)
 
--------------------------------------------------------------
+
+## Objective
+
+The methods are used in, and referred from, the Test Case specifications as
+shortcuts for steps shared between Test Cases. A Test Case that makes use of any
+of the methods defined in this document must refer directly to the
+specific method or methods.
+
+[To top]
+
+
+## Scope
+
+This document holds version 2 of the set of methods. Version 1 is defined in
+[Methods]. Methods from version 1 will be replaced by methods from version 2 in
+all Test Case specifications.
+
+Before the transition all Test Cases specifications use version 1 ([Methods]).
+During the transition it will be stated in each specification if the
+Test Case uses methods from version 1 ([Methods]) or methods from version 2 (this
+document). When the transition is completed, the version 1 document will be
+removed.
+
+[To top]
+
+
+## Internal methods
+
+Methods, in this document, that are referred to as *internal* or
+*internal method* must not be referred to from the Test Case specifications.
+*Internal methods* may only be referred to from methods in this document. Test
+Case specifications can freely refer to the other methods.
+
+[To top]
+
 
 ## Method: Get parent zone
 
 ### Method identifier
-**Get-Parent-Zone** 
-
-*In general, this method replaces [Method1] in the old [Methods].*
+**Get-Parent-Zone**
 
 ### Objective
 
+In general, this method replaces [Method1] in the [Methods], version 1.
+
 Obtain the parent zone of the child zone to be tested and the name server
 IP addresses of the parent zone. Determine if it is possible to perform
 the testing of the child zone.
 
-This is an external method that can be used by test cases.
-
 This method must, in general, use the same algorithm as Test Case [BASIC01], but
 without the messages.
 
 ### Inputs
 
 * "Child Zone" - The name of the child zone.
 * "Root Name Servers" - The IANA [List of Root Servers].
-* "Test Type" - The test type with values "undelegated test" or 
+* "Test Type" - The test type with values "undelegated test" or
   "normal test".
 
-This method also inherits the inputs of method [Get-Undel-Data].
+This method also inherits the output of method [Get-Undel-Data]
+as input, when applicable.
 
 ### Prerequisite
 
 If *Test Type* is "undelegated test", then as specified in method
 [Get-Undel-Data].
 
-### Ordered description of steps to be taken to execute the method
+### Test procedure
```

Comment:
I suggest "Method procedure" or even better just "Procedure. After all this is not a test case.

Reply:
It does not say "Test case procedure". I think "Test procedure" works well.